Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -451,12 +451,15 @@ QTabBar::tab:focus {
 
 #projectButton > QFrame  {
     margin-top:6px;
+    min-width: 185px;
+    max-width: 210px;
 }
```

Comment:
When I tested this change with different widths the project tiles jumped around when building a project.  Let's be more explicit and add a specific height/width for the over all project tile (projectButton) instead.  This seems to prevent the tile from resizing at all in my testing.  Can you test this and see if it works for you?

```suggestion
#projectButton  {
    min-width: 210px;
    max-width: 210px;
    min-height: 310px;
    max-height: 310px;
}

#projectButton > Frame {
    margin-top:6px;
}
```